src/DeepNgRoot/Tests/Frontend/angular/health-checks/health.check.spec.js

```diff
@@ -24,4 +25,9 @@ describe('Health checks', function() {
   it('Module name is [ng-root]"', function() {
     expect(moduleName).toBe('ng-root');
   });
+
+  it('Should load bootstrap', function() {
+    let actualResult = bootstrap();
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).